pom.xml

```diff
@@ -343,7 +343,7 @@
                         </oldVersion>
                         <parameter>
                             <onlyModified>true</onlyModified>
-                            <breakBuildOnBinaryIncompatibleModifications>false</breakBuildOnBinaryIncompatibleModifications> <!-- TODO: revert to true after v2.0.0 major release -->
+                            <breakBuildOnBinaryIncompatibleModifications>false</breakBuildOnBinaryIncompatibleModifications> <!-- TODO: revert to true after v3.0.0 major release -->
```

Comment:
Please create a JIRA for this so we don't forget it later :)

Reply:
https://oktainc.atlassian.net/browse/OKTA-348852

Reply:
done

Reply:
https://oktainc.atlassian.net/browse/OKTA-348852 for record.